spec/factories/raw_event.rb

```diff
@@ -1,7 +1,10 @@
 include ActionDispatch::TestProcess
 
+path_to_file = "#{Rails.root}/spec/fixtures/rails4_raw_data.json"
+
 FactoryGirl.define do
   factory :raw_event do
-    sequence(:data) { fixture_file_upload("#{Rails.root}/spec/fixtures/rails4_raw_data.json", 'application/json') }
+    association :application
+    sequence(:data) { JSON.parse(File.read(path_to_file))["data"].to_json }
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.